src/lib/components/islands/ChannelInfo.svelte

```diff
@@ -0,0 +1,81 @@
+<script lang="ts">
+    import type { getChannelInfo } from '$lib/server/services/content';
+    import * as Avatar from '$lib/shadcn/components/ui/avatar/index';
+    import Button from '$lib/shadcn/components/ui/button/button.svelte';
+    import * as Card from '$lib/shadcn/components/ui/card';
+    import ScrollArea from '$lib/shadcn/components/ui/scroll-area/scroll-area.svelte';
+    import Separator from '$lib/shadcn/components/ui/separator/separator.svelte';
+
+    export let channel: Awaited<ReturnType<typeof getChannelInfo>>;
```

Comment:
Would be nice to have a named type here.

Reply:
Added. I added it in a few other spots as well.